src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -129,6 +131,31 @@ public async Task BulkDocs()
             }
         }
 
+        private static void ValidateCommand(MergedBatchCommand command)
+        {
+            foreach (var document in command.ParsedCommands)
+            {
+                switch (document.Type)
+                {
+                    case CommandType.PUT:
+                    case CommandType.DELETE:
```

Comment:
Do we need it for anything but `PUT` ?

Reply:
Don't think so. 
no silent merges if the document is deleted.